packages/web-fragments/tsconfig.base.json

```diff
@@ -14,10 +14,10 @@
 		"strict": true,
 		"forceConsistentCasingInFileNames": true,
 		"module": "ESNext",
-		"moduleResolution": "Bundler",
+		"moduleResolution": "bundler",
 		"resolveJsonModule": true,
 		"isolatedModules": true,
 		"jsx": "react-jsx",
-		"types": ["vite/client"]
+		"types": ["vite/client", "node"]
```

Comment:
this is not right. we should be adding this to gateway specific tsconfig and not the base

Reply:
resolved